PYME/misc/check_for_updates.py

```diff
@@ -99,6 +99,9 @@ def check_for_updates(gui=True, force=False):
                 
         else:
             update_available = False
+            
+            if gui:
+                gui_prompt_no_update()
```

Comment:
Calling it here will pop up a "you are running the latest version" popup for all the automatic checks as well - this is not what we want - we only want to pop up a "no updates" dialog if you specifically run help->check for updates.